app/filters.py

```diff
@@ -3,8 +3,10 @@ def datetime_format(value, format="%d/%m/%y"):
 
 
 def snake_case_to_human(word):
-    return word.replace("_", " ").title()
+    if word:
+        return word.replace("_", " ").title()
 
 
 def kebab_case_to_human(word):
-    return word.replace("-", " ").title()
+    if word:
+        return word.replace("-", " ").title()
```

Comment:
May be worth a comment/docstring on this to note that this shouldn't be used to unslugify as "-" are used to replace other special characters too

Reply:
great shout